src/Deprecated11/Announcer.extension.st

```diff
@@ -0,0 +1,24 @@
+Extension { #name : #Announcer }
+
+{ #category : #'*Deprecated11' }
```

Comment:
Maybe this should go to a new Deprecated12 package 🤔

Reply:
Yes because Deprecated11 was removed :)